libraries/pbrlib/genglsl/lib/mx_microfacet_specular.glsl

```diff
@@ -247,6 +249,17 @@ float mx_fresnel_dielectric(float cosTheta, float ior)
     return 0.5 * x * x * (1.0 + y * y);
 }
 
+// https://renderwonk.com/publications/wp-generalization-adobe/gen-adobe.pdf
+vec3 mx_fresnel_hoffman_schlick(float cosTheta, vec3 F0, vec3 F82, vec3 F90, float exponent)
+{
+    const float COS_THETA_MAX = 1.0 / 7.0;
+    const float COS_THETA_FACTOR = 1.0 / (COS_THETA_MAX * pow(1.0 - COS_THETA_MAX, 6.0));
```

Comment:
For consistency you could use the added mx_pow6 function here as well.

Reply:
That's a good note, @niklasharrysson, and I had wanted to use the approach you describe, but some GLSL compilers require that a `const` value be defined entirely in terms of constant expressions.

So I actually see a compile-time error in my GLSL environment if I use the non-const `mx_pow6` as a sub-expression of COS_THETA_FACTOR.

For simplicity, I went back to using the built-in `pow` function, but there may be a more clever way of notating `mx_pow6` so that it can also be used in this way.

Reply:
Ah, that's fine, this was just a tiny thing I noticed.